### PR TITLE
Add docs for relative-deps

### PR DESCRIPTION
**Pull request by SaraVieira**

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -26,6 +26,16 @@ If you want to contribute and don't know where to start, here is a step-by-step
 6. Run `yarn` - install dependencies
 7. Run `yarn dev` - URL will be logged
 
+
+## Run designer and sdk in parallel
```
Comment (kof) [inline docs/contributing.md:30]: "Run" is probably not very clear in this context, you mean more "develop designer and SDK" or "watch changes"

File: docs/contributing.md [hunk 2]
```diff
@@ -26,6 +26,16 @@ If you want to contribute and don't know where to start, here is a step-by-step
...
+
+For this we current use [relative-deps](https://github.com/mweststrate/relative-deps) and the way you can run both things is the following:
+
+* Git clone the sdk at https://github.com/webstudio-is/webstudio-sdk
+* In the designer run `yarn dev` and in another window run `yarn sdk:watch`
```
Comment (kof) [inline docs/contributing.md:35]: watch:sdk

File: docs/contributing.md [hunk 3]
```diff
@@ -26,6 +26,16 @@ If you want to contribute and don't know where to start, here is a step-by-step
...
+
+This will trigger a build of the sdk when you make changes and that will in turn reload your designer.
```
Comment (kof) [inline docs/contributing.md:37]: "make changes in sdk" (not clear that the change you will be able to make is in SDK )

**Review by kof — APPROVED** [review]
approving in advance, a few wording things need fixes

File: docs/contributing.md (hunk 1 above)
Comment (SaraVieira) [reply docs/contributing.md]: Changed

File: docs/contributing.md (hunk 2 above)
Comment (SaraVieira) [reply docs/contributing.md]: changed

File: docs/contributing.md (hunk 3 above)
Comment (SaraVieira) [reply docs/contributing.md]: Updated.

Looks better?

**Pull request merged**